src/web/components/SmallHeadline.tsx

```diff
@@ -72,16 +72,14 @@ const linkStyles = css`
     }
 `;
 
-type HeadlineLinkSize = 'tiny' | 'xxsmall' | 'xsmall';
-
 type Props = {
     headlineString: string; // The text shown
     pillar: Pillar; // Used to colour the headline (dark) and the prefix (main)
     underlined?: boolean; // Some headlines have an underlined style
     showUnderline?: boolean; // Some headlines have text-decoration underlined when hovered
     prefix?: PrefixType;
     showQuotes?: boolean; // When true the QuoteIcon is shown
-    size?: HeadlineLinkSize;
+    size?: SmallHeadlineSize;
     coloured?: boolean; // When coloured, the headline takes the dark pillar colour
     linkTo?: string; // If provided, this turns the headlineString into an a tag
 };
```

Comment:
Ha, my next PR is this: https://trello.com/c/uxuAzr7q/886-rename-prefix-kicker

Very happy to introduce `trailImage`. In general, I'm trying quite hard to add the correct, design based nomenclature instead of my, guess based, approach

Reply:
![tenor-103841310](https://user-images.githubusercontent.com/638051/69260108-3f787100-0bb7-11ea-92da-e4ef1633a92f.gif)